RPS_ver2.py

```diff
@@ -38,22 +44,25 @@ def decide_winner(p1, p2):
 while True:
 
     # gather the player choices and convert to lowercase for easy comparison later
-    player1 = raw_input('Player one.. Rock, Paper, or Scissors? ').lower()
-    player2 = raw_input('Player two.. Rock, Paper, or Scissors? ').lower()
+    player_one_input = raw_input(rps_string.format("One")).lower()
```

Comment:
Use variable names that are more expressive than just 'player1' - it's one thing to use them up above, because it's literally the string that says "Player One", but it's quite another to use that down here.

Also make note of the rps_string - we defined that up at the top, and then we called the format function on it.  By passing in the value "One" and "Two", it substitutes those in place of {0}.  If we wanted to extend that to something like:

'''rps_string = "Player {0} has defeated Player {1}"
rps_string.format(winner, loser)'''

It would substitute those for us.